config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   root controller: :welcome, action: :index
+  get 'dashboard', to: 'welcome#dashboard'
+  resources :group, only: [:new, :create, :show]
```

Comment:
in line with the pluralize comment from before. Use plural form when defining routes for your resources

Model: `Group`
Controller: `GroupsController`
routes: `resources: groups`